patches/server/0181-check-whatever-Bukkit-getWorld-is-null.patch

```diff
@@ -0,0 +1,150 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: mohammed jasem alaajel <xrambad@gmail.com>
+Date: Sat, 20 Aug 2022 10:16:05 +0400
+Subject: [PATCH] check whatever Bukkit#getWorld is null
+
+
+diff --git a/src/main/java/puregero/multipaper/MultiPaper.java b/src/main/java/puregero/multipaper/MultiPaper.java
+index 0428f7d8702c8f1e837a9a95d31dc501374880bf..aa42f350731d6d6aaab76047733867d9f2efe1b9 100644
+--- a/src/main/java/puregero/multipaper/MultiPaper.java
++++ b/src/main/java/puregero/multipaper/MultiPaper.java
+@@ -147,7 +147,7 @@ public class MultiPaper {
+                 level.saveIncrementally(true);
+             }
+         }
+-        
```

Comment:
Please remove unintended formatting changes

Reply:
not sure where these are comming from didn't add single lines.

Reply:
fixed the format